docs/installing/compiler.md

```diff
@@ -21,53 +21,60 @@ To obtain the newest package:
+    needs to be next to the project folder.
+  - Create a folder .vscode in your project
+  - Create a file .vscode/cmake-kits.json with the following content. *IMPORTANT:* If you use the MingGW toolchain, choose `MinGW Makefiles` as preferred generator, for MSVC choose `Ninja`.
+  ```json5
+  [
+    {
+      "name": "GCC Bartman m68k",
+      "toolchainFile": "${workspaceFolder}\\..\\AmigaCMakeCrossToolchains\\m68k-bartman.cmake",
+      "environmentVariables": {
+        "PATH": "${env:BARTMAN_PREFIX_PATH}/opt/bin;${env:BARTMAN_PREFIX_PATH};${env:BARTMAN_PREFIX_PATH}/opt/bin/m68k-amiga-elf/bin;${env:PATH}"
+      },
+      "preferredGenerator": {
+        "name": "MinGW Makefiles"
+      },
+      "cmakeSettings": {
+        "CMAKE_C_COMPILER:FILEPATH": "${env:BARTMAN_PREFIX_PATH}/opt/bin/m68k-amiga-elf-gcc.exe",
+        "M68K_CPU": "68000",
+        "TOOLCHAIN_PREFIX": "m68k-amiga-elf",
+        "TOOLCHAIN_PATH": "${env:BARTMAN_PREFIX_PATH}/opt",
+        "CMAKE_AR": "${env:BARTMAN_PREFIX_PATH}/opt/m68k-amiga-elf/bin/ar.exe"
       },
-      {
-        "name": "GCC Bartman m68k",
-        "toolchainFile": "C:\\PATH_TO\\AmigaCMakeCrossToolchains\\m68k-bartman.cmake",
-        "compilers": {
-          "C": "C:\\Users\\YOUR_USER_NAME\\.vscode\\extensions\\bartmanabyss.amiga-debug-1.1.0-preview33\\bin\\opt\\bin\\m68k-amiga-elf-gcc.exe",
-          "CXX": "C:\\Users\\YOUR_USER_NAME\\.vscode\\extensions\\bartmanabyss.amiga-debug-1.1.0-preview33\\bin\\opt\\bin\\m68k-amiga-elf-g++.exe"
-        },
-        "environmentVariables": {
-          "PATH": "C:/Users/YOUR_USER_NAME/.vscode/extensions/bartmanabyss.amiga-debug-1.1.0-preview33/bin/opt/bin;C:/Users/YOUR_USER_NAME/.vscode/extensions/bartmanabyss.amiga-debug-1.1.0-preview33/bin;${env:PATH}"
-        },
-        "preferredGenerator": {
-          "name": "MinGW Makefiles"
-        },
-        "cmakeSettings": {
-          "M68K_CPU": "68000",
-          "TOOLCHAIN_PREFIX": "m68k-amiga-elf",
-          "TOOLCHAIN_PATH": "C:/Users/YOUR_USER_NAME/.vscode/extensions/bartmanabyss.amiga-debug-1.1.0-preview33/bin/opt"
-        },
-        "keep": true
-      }
-    ]
-    ```
-
-After this you should be good to go.
+      "keep": true
+    }
+  ]
+  ```
+  - Create a file .vscode/settings.json with the following content, replacing the extension name/version so that it matches your filesystem:
+  ```json5
+  {
+    "cmake.environment": {
+        "BARTMAN_PREFIX_PATH": "${userHome}/.vscode/extensions/bartmanabyss.amiga-debug-1.6.7/bin/win32/"
+    }
+  }
+  ```
+  - Create empty CMakeLists.txt file **and restart the editor** so that it can discover that you're inside CMake-based project.
+    (You can do this via command palette (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and typing "Reload Window".)
+  - Enter command palette (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and start typing "CMake Scan for kits" and select the Bartman kit from the options.
```

Comment:
I don't think that scanning for kits is okay here. Since you manually create the json file beforehand, you shouldn't need to scan for everything, just hit the "Configure Project" command.

Reply:
True, I just tried it again, you don't need to scan at all.